com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.cs

```diff
@@ -4435,6 +4505,24 @@ void ClearBuffers(HDCamera hdCamera, CommandBuffer cmd)
                     }
                 }
 
+#if ENABLE_VIRTUALTEXTURES
+                using (new ProfilingScope(cmd, ProfilingSampler.Get(HDProfileId.VTFeedbackClear)))
+                {
+                    RTHandle alreadyCleared = null;
```

Comment:
I don't understand this code, we are suppose to have a single buffer for both forward and deferred, it must be the same, so we should always clear it. There is something wrong here.
Note: we can have both deferred and forward at the same time in HDRP (can have hair and fabric render in forward and lit shader render in deferred)